packages/frontend/src/components/RichTextEditor/index.tsx

```diff
@@ -163,10 +169,33 @@ const Editor = ({
     },
     [editor],
   )
+  const {
+    isOpen: isSuggestionsOpen,
+    onOpen: openSuggestions,
+    onClose: closeSuggestions,
+  } = useDisclosure()
 
   return (
-    <Popover gutter={0} matchWidth={true}>
-      <div className="editor">
+    <Popover
+      gutter={0}
+      matchWidth={true}
+      isOpen={isSuggestionsOpen}
+      autoFocus={false}
+    >
+      <div
+        className="editor"
+        onClick={openSuggestions}
+        onFocus={openSuggestions}
+        onBlur={(event) => {
+          // Focus might shift to menu bar or other children, where we do _not_
+          // want to close our popper.
+          if (event.currentTarget.contains(event.relatedTarget)) {
+            return
+          }
```

Comment:
so this occurs when i click on the editor toolbar?

Reply:
TBH in prod right now you won't see this, since clicking anywhere inside the editor will close popper <.<